app/views/content_items/case_study.html.erb

```diff
@@ -40,7 +40,7 @@
     <% if @content_item.image %>
       <figure class="sidebar-image">
         <img src="<%= @content_item.image["url"] %>" alt="<%= @content_item.image["alt_text"] %>">
-        <figcaption><%= @content_item.image["caption"] %></figcaption>
+        <figcaption><%= @content_item.image["caption"].presence %></figcaption>
```

Comment:
This should really be a guard around the `<figcaption>` if there is no caption rather than rendering an empty one.